### DISCUSSION TITLE
add new language

### DISCUSSION THREAD

**Discussion by arezki1990** [wagtail/wagtail-localize#457] [Q&A]
Hello ,
Thanks for the great job, 
 i would like to add a new language ([Berber languages](https://en.wikipedia.org/wiki/Berber_languages)) which is not provided by django and wagtail-localize, how can i do that , 

thanks in advance

**Comment by kaedroho** [comment]
Hi @arezki1990 
Thanks!


You can add any extra language codes you like to the ``LANGUAGES`` and ``WAGTAIL_CONTENT_LANGUAGES`` settings. Both of these have the same format so you can set them to the same value:

```
LANGAUGES = WAGTAIL_CONTENT_LANGUAGES = [
     ('ber', "Berber languages"),

     # Any other languages you like to support here
]
```
